src/lib/icons/IconLockOpen.svelte

```diff
@@ -1,9 +1,13 @@
 <!-- source: https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Alock_open%3A -->
+<script lang="ts">
+  export let size = `18px`;
```

Comment:
the `DEFAULT_ICON_SIZE` is 20, it has to be 18?

Reply:
I know. I didn't want to change the size in this PR. Just add the prop to configure it.

If we change this size, I guess we'll need to change others that go in the same place like the normal lock clock for example.

Reply:
Makes sense